skimage/morphology/convex_hull.py

```diff
@@ -106,6 +106,13 @@ def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10):
     .. [1] https://blogs.mathworks.com/steve/2011/10/04/binary-image-convex-hull-algorithm-notes/
 
     """
+    image_shape = np.array(image.shape)
+    # Look for 1D axes
+    axes_1D = np.where(image_shape == 1)[0]
+    label_is_2D = (axes_1D.size != 0) and (len(image_shape) == 3)
+    if label_is_2D:
+        # If there is at least one axis 1D and image is 3D, treat it as 2D
+        image = np.squeeze(image, axis=axes_1D[0])
```

Comment:
After investigating some more, I think we can formulate this fix a bit simpler and hopefully more robust and universal:

```suggestion
    original_shape = np.array(image.shape)
    # Squeeze axes of length 1 such that original shape is at least 2D
    # in which case possible_hull returns coords that ConvexHull can deal with
    image = image.squeeze()
    if image.ndim < 2:
        image = np.atleast_2d(image)
```

This seems to work for all cases that we discussed.

@stefanv, concerning the progress: I hope that this is the fix is the silver bullet. :crossed_fingers: But there is some work on new and more extensive tests left to do. I'd like to see at least all cases that we discussed in this PR as test. 

@zoccoler once your setup works, would you be okay with adding these :pray:? Let me know if you need some clarification or don't agree with something.

Reply:
> `np.atleast_2d`

Hey I didn't know that one, nice @lagru! It should work!

> @zoccoler once your setup works, would you be okay with adding these 🙏? 

Sure! So, to clarify, there's need to:

- [ ] move/adapt tests from `measure/tests/test_regionprops.py` to `morphology/tests/test_convex_hull.py` which include:
  - [ ] convex hull of 2d label in 3d image
  - [ ] convex hull of 1d label (straight line) in 3d image
- [ ] add a minimal test to `measure/tests/test_regionprops.py` similar to #6432 description

Please feel free to add if something is missing.